flambe/field/text.py

```diff
@@ -97,6 +149,19 @@ def __init__(self,  # nosec
             embedding matrix. Defaults to False.
 
         """
+        if embeddings:
+            if model:
+                raise ValueError("Cannot submit a model and use the embeddings parameters" +
+                                 "simultaneously. Use the 'from_embeddings' factory instead.")
+
+            warnings.warn("The embeddings parameters will be deprecated in future release. " +
```

Comment:
Should the warning include `setup_all_embeddings` or is it obvious?

Reply:
Not only `setup_all_embeddings` but also other embeddings-specific parameters will go away. Adding this to the warning.